src/heitech.zer0mqXt.core/Transport/Message.cs

```diff
@@ -91,7 +104,8 @@ internal static XtResult<TMessage> ParseRqRepMessage<TMessage>(this NetMQMessage
             bool isSuccess = Convert.ToBoolean(configuration.Serializer.Deserialize<string>(successFrame));
             if (!isSuccess)
             {
-                var excptn = new InvalidOperationException("Frame2 holds " + configuration.Serializer.Encoding.GetString(successFrame) + " therefore the Message cannot be savely parsed and is disregarded instead");
+                var exceptnText = configuration.Serializer.Deserialize<string>(payloadFrame);
+                var excptn = new ZeroMqXtSocketException("Server failed with" +  exceptnText);
```

Comment:
should I complain about the abbreviations? ... nah ... I just leave it (... and yet I have still complained 😉)

Reply:
Yes you should, and you did (:. I will fix it.